setup.md

```diff
@@ -38,6 +38,7 @@ In this workshop, we will be moving code from a Jupyter notebook into a Python p
             │       └── water.xyz
             └── starting_notebook.ipynb
     ~~~
+- Please ignore any `.DS_Store` files included in the directiories, they are nothing but remnants from MacOS.
```

Comment:
The `.DS_Store` file is present because these were zipped from a mac. I agree that having this file in the folder is not ideal but we are planning in the next release to upload a data.zip without this file. Since you have found this error, it may be nice to fix it at this point instead of adding this sentence only to remove it when we review the materials next month.